AppCenterDistribute/AppCenterDistribute/MSACDistribute.h

```diff
@@ -65,14 +66,14 @@ typedef NS_ENUM(NSInteger, MSACUpdateTrack) {
  *
  * @param apiUrl The new URL.
  */
-+ (void)setApiUrl:(NSString *)apiUrl;
++ (void)setApiUrl:(NSString *)apiUrl NS_SWIFT_NAME(set(apiUrl:));
 
 /**
  * Change the base URL that is used to install update.
  *
  * @param installUrl The new URL.
  */
-+ (void)setInstallUrl:(NSString *)installUrl;
++ (void)setInstallUrl:(NSString *)installUrl  NS_SWIFT_NAME(set(installUrl:));
```

Comment:
I guess it should be property like in other places. `set(installUrl:)` signature looks wierd

Reply:
I agree, that `set(installUrl:)` looks a bit weird, but I don't think we can make it property. There is no getter available for `installUrl`. Do you think it's worth updating public API and expose the getter?

Or do we just leave it without any changes?

Reply:
Adding getter is OK for it